VAGRANT.md

```diff
@@ -44,3 +47,7 @@ rails server --binding=0.0.0.0
 You edit the code on your computer using the code editor you are used to using, then through shared folders the code is updated on the VM instantly.
 
 You should run the tests before submitting any patch or Pull Request back to the original repository. Please see [CONTRIBUTING.md](CONTRIBUTING.md) for more information.
+
+## Troubleshooting
+
+If you have issues installing ssh tunnel to the environment, try adding `:nfs_udp => false` to `override.vm.synced_folder` options in the `Vagrantfile`.
```

Comment:
So what you're saying is that we don't understand why this fixes the problem, it's just a "cargo cult" fix from an internet search?

Can you give more details of the error that occurs when you don't do this?